fix(tdengine): update the default template and fix a typo in the changes (#11286)
emqx/emqx: changes/ee/fix-11266.en.md

   `insert into table_${topic} values (${ts}, '${id}', '${topic}')`

> Note: This is a breaking change, at the former, the string-type values are automatically quoted, but now, you should manually quote them.

Reviewer: ```suggestion
Note: This is a breaking change. Previously the values of string type were quoted automatically, but now they must be quoted explicitly.
```
